### PR TITLE
ADM-348 T5C - [frontend] [Config] - Source control section

### PR DESCRIPTION
**Pull request by tingwang1234**
1. change error message of SouceControl and PipelineTool components when token is invalid
2. fix unit tests

### CODE REVIEW

File: frontend/__tests__/src/fixtures.ts [hunk 1]
```diff
@@ -18,6 +18,10 @@ export const RESET = 'Reset'
 
 export const VERIFIED = 'Verified'
 
+export const mockInfo = 'mockToken'
```
Comment (guzhongren) [inline frontend/__tests__/src/fixtures.ts:21]: Constant?

**Review by fengxinhou — APPROVED** [review]
LGTM

File: frontend/__tests__/src/fixtures.ts (hunk 1 above)
Comment (tingwang1234) [reply frontend/__tests__/src/fixtures.ts]: have changed this constant to MOCK_INFO

**Review by ZhiqiaoYang — APPROVED** [review]
LGTM

File: frontend/__tests__/src/fixtures.ts (hunk 1 above)
Comment (tingwang1234) [reply frontend/__tests__/src/fixtures.ts]: I'm sorry to misunderstand this problem, I'll create a new PR to solve it soon.

**Pull request merged**